demo/app/index.html

```diff
@@ -229,7 +229,8 @@
     </div>
 	</footer>
   <script src="../../publish/skylink.complete.js"  type="text/javascript"></script>
-	<script src="../config.js"   type="text/javascript"></script>
+	<script src="../edge-attachmediastream.js"   type="text/javascript"></script>
```

Comment:
Hmm this is strange because on my sublime editor it appears no extra indentation.